v4/config_file.md

```diff
@@ -6,9 +6,7 @@ parent: v4
 ---
 
 # Config file
-
-Before running PoracleJS for the first time, you need to create a config file. There is an example you can copy over to begin with `cp config/default.json config/local.json`.
-
+The configuration file is how PoracleJS determines how to connect to services and how to configure itself during runtime. The following sections each describe a particular sub-section in the configuration file. After making changes to the file we should ensure it is valid JSON. If it is not valid JSON PoracleJS will not start. You can use [JSONLint](https://jsonlint.com/) to ensure the file is valid JSON. It will attempt to highlight any issues that would cause the file to be invalid JSON.
```

Comment:
Does JSONlint mind the fact that you can include comments (and we do now in default.json)

Reply:
JSONLint does not like comments. I will add a note stating that

Reply:
I think the node-config library that we are using supports json5 not just cjson.
But anyway, I found this -- https://codebeautify.org/json5-validator -- which will online validate json with comments. There may be some things it accepts which node-config does not I couldn't find a definitive reference to say what parsing style node-config uses at a quick search but at least if it passes this it won't have any obvious things wrong with it!

Reply:
Yea I was searching last night for cJSON validator and I cannot find one. I will just reference using your browsers developer tools / console and hoping for the best :)